### ISSUE TITLE
`ValidationError` for `CalculatorCharacterDetails`: `none` is not an allowed value for field `weapon`

### ISSUE THREAD

**Issue by seriaati** [seriaati/genshin.py#117]
Sentry issue link: https://seria-ati.sentry.io/share/issue/e0cc00126dbf41679c37fc22fbd82604/  
Could be an edge case considering that I only received this error once.

**Comment by ashleney** [comment]
Yeah, that's really weird, I don't think a character can lack a weapon. I'll just keep this open in case someone experiences a similar issue.
